set.mm

```diff
@@ -554884,19 +555102,20 @@ relation with an intersection with the same Cartesian product (cf.
       ABEFMN $.
   $}
 
-  $( Please delete when ~ opidg moves from a Mathbox to the main set.mm .
-     (Contributed by Peter Mazsa, 22-Jul-2019.) $)
-  opidORIG $p |- ( A e. V -> <. A , A >. = { { A } } ) $=
-    ( wcel cop csn cpr wceq dfopg anidms dfsn2 eqcomi preq2i eqtr4i syl6eq ) AB
-    CZAADZAEZAAFZFZQEZOPSGAABBHISQQFTRQQQRAJKLQJMN $.
-
   $( Equality conditions for ordered pairs ` <. A , A >. ` and
-     ` <. B , B >. ` .  (Contributed by Peter Mazsa, 22-Jul-2019.) $)
-  opideq $p |- ( ( A e. V /\ B e. W ) ->
+     ` <. B , B >. ` .  (Contributed by Peter Mazsa, 22-Jul-2019.)  (Revised by
+     AV, 16-Feb-2022.) $)
+  opideq $p |- ( A e. V -> ( <. A , A >. = <. B , B >. <-> A = B ) ) $=
+    ( wcel cop wceq wa wb opthg anidms anidm syl6bb ) ACDZAAEBBEFZABFZOGZOMNPHA
+    ABBCCIJOKL $.
+
+  $( Obsolete version of ~ opideq as of 16-Feb-2022.  (Contributed by Peter
+     Mazsa, 22-Jul-2019.)  (Proof shortened by AV, 16-Feb-2022.)
+     (New usage is discouraged.)  (Proof modification is discouraged.) $)
+  opideqOLD $p |- ( ( A e. V /\ B e. W ) ->
```

Comment:
I would prefer to delete this opideqOLD: it seem silly after opideq and nobody needs it (incl. me).

Reply:
Usually, OLD theorem will be deleted after one year. However, I can delete it immediatly. But let's wait for other opinions.

Reply:
OK to delete now.